paf-mvp-client-express/src/client-node.ts

```diff
@@ -56,127 +56,156 @@ export class ClientNode extends Node {
     // *****************************************************************************************************************
     this.app.expressApp.get(
       jsonProxyEndpoints.read,
+      this.startSpan({
```

Comment:
Note: I planned to move the call to `startSpan` in another PR.
But in this PR we start using data in other handlers, that was set in `startSpan` so it **has** to be called first.